README.md

```diff
@@ -4,13 +4,60 @@ HasCore: HasGeek API
 ![Build status](https://secure.travis-ci.org/hasgeek/hascore.png)
 [![Coverage Status](https://coveralls.io/repos/hasgeek/hascore/badge.png?branch=master)](https://coveralls.io/r/hasgeek/hascore?branch=master)
 
-Hascore provides helper API methods including the networkbar that is shown
-on all sites, and API endpoints for geolocation and language analysis.
+Hascore provides helper API methods including the networkbar that is shown on all sites, and API endpoints for geolocation and language analysis.
 
-Hascore requires NLTK. You will need to install NLTK's corpus data.
-Be warned, this is about 1.8 GB:
+-----
+### NLTK
+
+Hascore requires NLTK. You will need to install NLTK's corpus data. Be warned, this is about 1.8 GB:
 
     $ python -m nltk.downloader all
+    
+### Postgres
```

Comment:
Club the sections NLTK and Postgres and relevantly title it, such as "Setup dependencies". Under that use "__" (markdown syntax to denote text in bold face) to highlight NLTK and Postgress specific instructions